src/elastio-snap.c

```diff
@@ -5211,12 +5211,14 @@ static int bdev_switch_ownership(const char __user *dir_name, int follow_flags,
 	struct path path = {};
 	struct snap_device *dev;
 	struct block_device *bdev;
+	char bdev_name[BDEVNAME_SIZE];
```

Comment:
do you need to add `MAYBE_UNUSED(bdev_name)` to avoid warnings on the compilation?